### PR TITLE
XIVY-14357 add options of all known namespaces

### PR DESCRIPTION
**Pull request by ivy-lgi**
any known namespace can now be selected in the combobox
a new namespace can be input as well

added unittests for gathering all known namespaces
improved playwright test to check that known namespaces are present in the selection of the combobox

### CODE REVIEW

**Comment by ivy-lgi** [comment]
namespace-options

File: packages/variable-editor/src/components/variables/master/AddVariableDialog.css [hunk 1]
```diff
@@ -1,3 +1,11 @@
 .add-variable-dialog-trigger-button {
   margin-left: auto;
 }
+
+.ui-combobox-menu {
+  z-index: 50;
+}
```
Comment (ivy-lli) [inline packages/variable-editor/src/components/variables/master/AddVariableDialog.css:7]: I think this should not be needed any longer. 
Try to update to the latest ui-components.

File: packages/variable-editor/src/components/variables/master/AddVariableDialog.css [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+
+.ui-combobox-menu:empty {
+  display: none;
+}
```
Comment (ivy-lli) [inline packages/variable-editor/src/components/variables/master/AddVariableDialog.css:11]: needed?

File: packages/variable-editor/src/components/variables/master/AddVariableDialog.tsx [hunk 3]
```diff
@@ -73,7 +79,7 @@ export const AddVariableDialog = ({ table, variables, setVariables, setSelectedV
             <Input />
```
Comment (ivy-lli) [inline packages/variable-editor/src/components/variables/master/AddVariableDialog.tsx:79]: Maybe not part of this pr, but a very good gif. We should not allow an empty name, but show a message if someone tries to add a variable with no name

File: packages/variable-editor/src/components/variables/master/AddVariableDialog.css (hunk 1 above)
Comment (ivy-lgi) [reply packages/variable-editor/src/components/variables/master/AddVariableDialog.css:7]: You are right, but I notice that after updating the toolbar also stays while the dialog is open :sweat_smile: 
image

File: packages/variable-editor/src/components/variables/master/AddVariableDialog.css (hunk 2 above)
Comment (ivy-lgi) [reply packages/variable-editor/src/components/variables/master/AddVariableDialog.css:11]: Not necessarily, but I think it works better. If I remove it, things like this seem cumbersome:
hide-empty-menu

File: packages/variable-editor/src/components/variables/master/AddVariableDialog.tsx (hunk 3 above)
Comment (ivy-lgi) [reply packages/variable-editor/src/components/variables/master/AddVariableDialog.tsx:79]: Going to add validation with another PR :+1:

File: packages/variable-editor/src/components/variables/master/AddVariableDialog.css (hunk 1 above)
Comment (ivy-lli) [reply packages/variable-editor/src/components/variables/master/AddVariableDialog.css]: :(
sadly this has also a zindex... Need to remove this. Hopefully which no side effect :D

**Review by ivy-lli — APPROVED** [review]
